src/pt/lycantoons/src/eu/kanade/tachiyomi/extension/pt/lycantoons/LycanToons.kt

```diff
@@ -0,0 +1,190 @@
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import keiyoushi.utils.jsonInstance
+import keiyoushi.utils.parseAs
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.intOrNull
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import okhttp3.Response
+import org.jsoup.Jsoup
+
+private val TAG = "LycanToons"
+
+class LycanToons : HttpSource() {
+
+    override val name = "Lycan Toons"
+
+    override val baseUrl = "https://lycantoons.com"
+
+    override val lang = "pt-BR"
+
+    override val supportsLatest = true
+
+    override val client = network.cloudflareClient
+
+    private val pageHeaders by lazy {
+        headers.newBuilder()
+            .add("Referer", "$baseUrl/")
+            .build()
+    }
+
+    // =====================Popular=====================
+
+    override fun popularMangaRequest(page: Int): Request = metricsRequest("popular", page)
+
+    override fun popularMangaParse(response: Response): MangasPage =
+        response.parseAs<PopularResponse>().data.toMangasPage()
+
+    // =====================Latest=====================
+
+    override fun latestUpdatesRequest(page: Int): Request = metricsRequest("recently-updated", page)
+
+    override fun latestUpdatesParse(response: Response): MangasPage =
+        response.parseAs<PopularResponse>().data.toMangasPage()
+
+    // =====================Search=====================
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val payload = SearchRequestBody(
+            limit = PAGE_LIMIT,
+            page = page,
+            search = query,
+            seriesType = filters.valueOrEmpty<SeriesTypeFilter>(),
+            status = filters.valueOrEmpty<StatusFilter>(),
+            tags = filters.selectedTags(),
+        )
+
+        val body = json.encodeToString(payload).toRequestBody(JSON_MEDIA_TYPE)
+        return POST("$baseUrl/api/series", headers, body)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage =
+        response.parseAs<SearchResponse>().series.toMangasPage()
+
+    override fun getFilterList(): FilterList = LycanToonsFilters.get()
+
+    // =====================Details=====================
+
+    override fun mangaDetailsRequest(manga: SManga): Request = seriesRequest(manga.slug())
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val result = response.parseAs<SeriesDto>()
+        return result.toSManga().apply { initialized = true }
```

Comment:
I don't think you need `initialized = true` here, that is already done in `HttpSource`:
https://github.com/mihonapp/mihon/blob/e8c6e3364dded0cb21df572be175cbb01adc243f/source-api/src/commonMain/kotlin/eu/kanade/tachiyomi/source/online/HttpSource.kt#L223-L229